spark/src/main/java/org/opensearch/sql/spark/transport/TransportGetJobRequestAction.java

```diff
@@ -0,0 +1,52 @@
+/*
+ *
+ *  * Copyright OpenSearch Contributors
+ *  * SPDX-License-Identifier: Apache-2.0
+ *
+ */
+
+package org.opensearch.sql.spark.transport;
+
+import org.opensearch.action.ActionType;
+import org.opensearch.action.support.ActionFilters;
+import org.opensearch.action.support.HandledTransportAction;
+import org.opensearch.common.inject.Inject;
+import org.opensearch.core.action.ActionListener;
+import org.opensearch.sql.spark.transport.model.GetJobActionRequest;
+import org.opensearch.sql.spark.transport.model.GetJobActionResponse;
+import org.opensearch.tasks.Task;
+import org.opensearch.transport.TransportService;
+
+public class TransportGetJobRequestAction
+    extends HandledTransportAction<GetJobActionRequest, GetJobActionResponse> {
+
+  public static final String NAME = "cluster:admin/opensearch/ql/jobs/read";
```

Comment:
user has permission on `cluster:admin/opensearch/ql/jobs/read` can access
1. `_plugins/_query/jobs` list all the job status.
2. `_plugins/_query/jobs/{jobId}` fetch specify job status.

Reply:
yes.